wormhole_attester/sdk/js/src/index.ts

```diff
@@ -75,27 +195,6 @@ export function getBatchSummary(batch: BatchPriceAttestation): string {
   return JSON.stringify(abstractRepresentation);
 }
 
-export async function getSignedAttestation(
```

Comment:
Is this function not used anywhere?

Reply:
ah no, i checked before deleting it. a removed component was using it.